package.json

```diff
@@ -26,7 +26,7 @@
     "@formatjs/cli": "^6.2.7",
     "@playwright/test": "^1.42.1",
     "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/react": "^14.0.0",
+    "@testing-library/react": "^16.0.1",
```

Comment:
Fix so that `act` from the library is not highlighted as deprecated is in a later version.
https://github.com/testing-library/react-testing-library/pull/1319